rp2040-hal/src/uart/writer.rs

```diff
@@ -123,6 +123,19 @@ pub(crate) fn disable_tx_interrupt(rb: &RegisterBlock) {
     });
 }
 
+/// A break on the current object
+pub struct Break<'a, T>(&'a mut T);
```

Comment:
This could be marked as `must_use`.
```suggestion
#[must_use]
pub struct Break<'a, T>(&'a mut T);
```